src/billing/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard.test.ts

```diff
@@ -193,9 +191,9 @@ describe('AzureBillingProjectWizard', () => {
       [],
       false
     );
+    await screen.findByText('Billing project name already exists');
```

Comment:
Wait for the UI to update with the fact that the billing project name exists _before_ verifying that create button is disabled.